### DISCUSSION TITLE
Creating an alert when the time between two events is greater than 30s

### DISCUSSION THREAD

**Discussion by Maimz** [grafana/grafana#37891] [Dashboards, Panels & Visualizations]
Hello Grafana community !
I am a new user of Grafana. My question is about the alerts conditions. 
I have two queries A and B. 
A -> Transmission Lost 
B -> Transmission Recovered. 

I would like to raise an alert when the count of query B isn't arriving in the 30seconds following the count of query A ? 
If the event arriving in the 30s after a "Transmission Lost" is not a "Transmission Recovered" then raise an alarm.

Is it possible ?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
